code_climate.rb

```diff
@@ -31,8 +31,11 @@ def self.repo_stats(repo_name)
     repo = cc.repo_from_github_slug(repo_name)
     return {} if repo.nil?
     
-    repo_id     = repo["id"]
-    snapshot_id = repo["relationships"]["latest_default_branch_snapshot"]["data"]["id"]
+    repo_id       = repo['id']
+    snapshot      = repo['relationships']['latest_default_branch_snapshot']
+    snapshot_data = snapshot['data']
```

Comment:
Should we use `fetch_path` or `dig` instead?
```
snapshot_id = repo.dig("relationships", "latest_default_branch_snapshot", "data", "id")
return {} if snapshot_id.nil?
```

Reply:
thanks!